aiohomekit/controller/ble/discovery.py

```diff
@@ -161,6 +174,7 @@ async def finish_pairing(pin: str) -> BlePairing:
             obj = self.controller.pairings[alias] = BlePairing(
                 self.controller,
                 pairing,
+                device=self.device,
```

Comment:
If we get disconnected before the next advertisement updates the device, everything blows up